.gitignore

```diff
@@ -3,3 +3,6 @@
 
 node_modules
 _site
+
+.idea
+*.docx
```

Comment:
Not a big issue, but sharing some advice I was given about `.gitignore` files; the one for the repo should be specific to files in the repo; other files that are specific to an individual user’s computer should be in a user’s own global configuration file. See https://docs.github.com/en/get-started/git-basics/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer